SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -46,4 +46,10 @@ namespace SpatialGDKServicesConstants
 	const FString DevLoginDeploymentTag = TEXT("dev_login");
 
 	const FString UseChinaServicesRegionFilename = TEXT("UseChinaServicesRegion");
+
+	const FString ProxyFileDirectory = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectIntermediateDir(), TEXT("Improbable")));
+	const FString ProxyInfoFilePath = FPaths::Combine(ProxyFileDirectory, TEXT("ServerReceptionistProxyInfo.json"));
+
+	const FString LsofCmdFilePath = TEXT("/usr/sbin/");
+	const FString KillCmdFilePath = TEXT("/bin/");
```

Comment:
move these into the `#elif PLATFORM_MAC` because the paths won't work on windows